web-marketplace/src/components/organisms/SellerSetupAccount/SellerSetupAccount.tsx

```diff
@@ -1,19 +1,22 @@
 import { Trans } from '@lingui/macro';
+import InfoTooltip from 'web-components/src/components/tooltip/InfoTooltip';
 
 import ContainedButton from 'web-components/src/components/buttons/ContainedButton';
 import Card from 'web-components/src/components/cards/Card';
 import { Title } from 'web-components/src/components/typography';
 
 import { useAuth } from 'lib/auth/auth';
+import { useWallet } from 'lib/wallet/wallet';
 
 import { useStripeAccount } from './hooks/useStripeAccount';
 
 const SellerSetupAccount = () => {
   const { activeAccount, privActiveAccount } = useAuth();
+  const { loginDisabled } = useWallet();
   const { setupAccount, openLoginLink } = useStripeAccount();
 
   return (
-    privActiveAccount?.can_use_stripe_connect && (
+    (privActiveAccount?.can_use_stripe_connect || loginDisabled) && (
```

Comment:
we should not show that if loginDisabled
the user has to have `can_use_stripe_connect` set to true, but we can't actually check that if loginDisabled since we do not have access to privActiveAccount in that case so we just can't display the tooltip below anyway
@erikalogie designed for this use case, but this isn't actually possible technically